features/window.yml

```diff
@@ -0,0 +1,30 @@
+name: Window
+description: "The Window object is home to a variety of functions, namespaces, objects, and constructors. Some of these are associated with the user interface window the document is loaded in, but many are items that need to be available in the global scope."
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-window-object
+status:
+  compute_from: api.Window
+compat_features:
+  - api.Window
+  - api.Window.DOMContentLoaded_event
+  - api.Window.blur_event
+  - api.Window.close
+  - api.Window.closed
+  - api.Window.document
```

Comment:
This could belong in `dom.yml`. Here's fine, but this is yet another case where overlapping keys would be nice.